extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/InfinispanClientRuntimeConfig.java

```diff
@@ -18,6 +18,12 @@
     @ConfigItem
     public Optional<String> serverList;
 
+    /**
+     * Configures hotrod protocol settings
+     */
+    @ConfigItem
+    public InfinispanHotRodClientRuntimeConfig hotrod;
```

Comment:
I would just say to add it as properties in this class directly, no reason to make another sub property for the runtime config

Reply:
The reason why it was moved to separate class was to have the prefix ‘hotrod’ in the config. Just for clarity in case there will be other approach to configure communication and to separate it from general Infinispan config params 

But if you prefer to have it all in single class I can refactor it

Reply:
I would say to keep it in the same class, all of these properties are the same level in the hotrod-client.properties file as well.